### PR TITLE
Instructional resources

### PR DESCRIPTION
**Pull request by agorina**
I am not sure what is the best way to handle it.
This is something that needs to be run AFTER asmts are loaded.

### CODE REVIEW

File: reporting/sql/Instructional_resources_dml.sql [hunk 1]
```diff
@@ -0,0 +1,25 @@
+-- loads data AFTER asmts have been loaded into reporting
+-- original source is : http://www.smarterbalanced.org/educators/the-digital-library/
+
+INSERT IGNORE INTO reporting.instructional_resource (id, resource)
+SELECT id, 'https://portal.smarterbalanced.org/library/en/v1.0/digital-library-connections-grade-3-number-and-operations-in-base-ten.docx' AS resource FROM reporting.asmt WHERE label = 'Grade 03 Math - Number and Operations in Base Ten (IAB)'
```
Comment (agorina) [inline reporting/sql/Instructional_resources_dml.sql]: Note that this returns multiple records (per grade).

**Comment by malaffoon** [comment]
I'd approve this but i have no idea what/why/where (pretty clear on how/who though).

**Comment by agorina** [comment]
@malaffoon , it is a phase 1 implementation of the Digital Library.
In short: we do not have requirements and SBAC does not really have a lot of resources built yet. As a shortcut, we are doing this. More can be found here: https://jira.fairwaytech.com/browse/DWR-287

File: reporting/sql/Instructional_resources_dml.sql (hunk 1 above)
Comment (bgsavage) [inline reporting/sql/Instructional_resources_dml.sql]: So `id` is `asmt.id` ?

Comment (agorina) [inline reporting/sql/Instructional_resources_dml.sql]: That is correct.

Comment (bgsavage) [inline reporting/sql/Instructional_resources_dml.sql]: Should we consider renaming this field to indicate so?

Comment (agorina) [inline reporting/sql/Instructional_resources_dml.sql]: Good point. We should.

**Pull request merged**